src/docs/Foundations/Colors/index.mdx

```diff
@@ -92,13 +92,13 @@ Use the [info]() swatch to highlight anything related to informational feedback
 
 ## Secondary Green
 
-Secondary Green is for success. In LEROY MERLIN case, it is exacltly the case as the primary-01. But other brands or BU should be able to change that.
+Secondary Green is for success. In LEROY MERLIN case, it is exactly the case as the primary-01. But other brands or BU should be able to change that.
```

Comment:
In LEROY MERLIN case, this color is exactly the case as `primary-01` but other brands or BU should be able to change that.